.java-version

```diff
@@ -0,0 +1 @@
+10.0
```

Comment:
Tenker vi bør holde oss på Java 8 da dette er open source.

Reply:
Ahh. Oki. Men skal man ikke slutte å komme med oppdateringer for Java 8 i januar 2019? Har lest så masse datoer og blitt passe forvirret, men nå er jo java 11 kommet og det er jo neste LTS-versjon.

Reply:
[Oracle Java SE](https://www.oracle.com/technetwork/java/javase/tech/eol-135779.html) sier januar 2019 for commercial use. Vi er vel under denne eller har jeg misforstått? Disse lisensgreiene er ikke mitt bord, men om det stemmer så er vi vel ikke veldig frempå om vi oppgraderer Java-versjon her nå?